.travis.yml

```diff
@@ -13,7 +13,6 @@ before_install:
 
 install:
```

Comment:
Yeah it does. See https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Travis-CI-uses-npm. We might have to add `language: node_js` at the top though.